packages/asset/test/AuthSuperValidator.test.ts

```diff
@@ -0,0 +1,111 @@
+import {ethers} from 'ethers';
+import {expect} from 'chai';
+import runSetup from './fixtures/authValidatorFixture';
+
+describe('AuthValidator, (/packages/asset/contracts/AuthValidator.sol)', function () {
+  describe('General', function () {
+    it('should assign DEFAULT_ADMIN_ROLE to the admin address from the constructor', async function () {
+      const {authValidatorAdmin, AuthValidatorContract} = await runSetup();
+      const DEFAULT_ADMIN_ROLE =
+        await AuthValidatorContract.DEFAULT_ADMIN_ROLE();
+      const hasRole = await AuthValidatorContract.hasRole(
+        DEFAULT_ADMIN_ROLE,
+        authValidatorAdmin.address
+      );
+      expect(hasRole).to.equal(true);
+    });
+    it('should allow admin to set signer for a given contract address', async function () {
```

Comment:
ideally we also have the reverse option, should not allow signer to be set if not admin

Reply:
True, will add